pulp_ansible/app/tasks/collections.py

```diff
@@ -720,6 +717,7 @@ async def _add_namespace(self, name, namespace_sha):
             namespace = AnsibleNamespaceMetadata(**namespace)
             dc = DeclarativeContent(namespace, d_artifacts=da)
             await self.put(dc)
+            await self.parsing_namespace_progress_bar.aincrement()
             return True
 
         return False
```

Comment:
We don't need the 
```suggestion
            await self.parsing_namespace_progress_bar.aincrement()

        log.info("Failed to find namespace...")
```
The `return True/False` is no longer needed, so maybe we add some log statements for when we fail to add the namespace metadata.